phpBB/styles/prosilver/theme/icons.css

```diff
@@ -27,6 +32,13 @@ blockquote cite:before,
 	text-rendering: auto; /* optimizelegibility throws things off #1094 */
 }
 
+.icon[class*="png-"],
+.icon[class*="svg-"] {
```

Comment:
do not do this check comment above. These should all have there own class `.svg` `.png` `.fa`. Furthermore do not worry about any changes to the styling that is outside the scope of this pr and will be handled when i convert everything in the templates over in a seperate pr as The whole file will be replaced. The svg style above would not really go in this file but at the top of common.css with the other resets hereh: https://github.com/phpbb/phpbb/blob/master/phpBB/styles/prosilver/theme/common.css#L150

Reply:
Ah right, I understand.
I'll just make those small changes to just the `icons.css` as it's rather small file anyways and allows me/us to properly test things and make sure they are all working as intended.

Reply:
You can make them locally just do not merge them to the pr

Reply:
Okay, I was going for doing it now. Test everything and then revert the PR in such a state so it only has the necessary stuff (twig extension and icon templates). This way other people can easily "test/see/fiddle" with it too.